### DISCUSSION TITLE
V1.1.0

### DISCUSSION THREAD

**Discussion by driftingly** [driftingly/rector-laravel#191] [General]
When should we release the next version?

Currently sitting at: 

* Adds a fix for ThrowIfRector causing issues with assigned vars by @peterfox in https://github.com/driftingly/rector-laravel/pull/181
* Adds the ThrowIFAndThrowUnlessExceptionsToUseClassStringRector rule by @peterfox in https://github.com/driftingly/rector-laravel/pull/180
* Validation Rule string value to Array by @peterfox in https://github.com/driftingly/rector-laravel/pull/183
* Allow RouteActionCallableRector to work with \Route calls by @GeniJaho in https://github.com/driftingly/rector-laravel/pull/185
* Drop renaming collection `every` to `nth` by @driftingly in https://github.com/driftingly/rector-laravel/pull/188
* Dispatch to DispatchSync with non queueable job by @peterfox in https://github.com/driftingly/rector-laravel/pull/182

@peterfox Do you want to wait for those last two PRs to be merged in?

- #184 
- #189

**Comment by driftingly** [comment]
It really bugs me I messed up a release and jumped from `0.30.0` to `0.40.0` instead of `0.31.0` 🤦

**Reply by GeniJaho** [reply]
I thought that was on purpose, what a boss move I thought 😂

**Reply by driftingly** [reply]
LOL... `0.31.0`?  Nah YOLO `0.40.0` 🚀

**Comment by peterfox** [comment]
@GeniJaho #184 is waiting on Rector to release a new version as it has dependencies on changes I made there.

**Comment by peterfox** [comment]
@ driftingly if we can get #192 and #194 in, it would then be good to do a v1.1.0 release. I want more time with #184 to clean up some of the set configs with a dedicated class to generate all the rule configs.

**Comment by GeniJaho** [comment]
Since the two PRs mentioned above have been merged, I think we should do a release.
Also https://github.com/driftingly/rector-laravel/issues/199.

**Comment by peterfox** [comment]
There's a bit of a problem currently. I can't track down the why but tests fail with one of the rules I've added in the past.

https://github.com/driftingly/rector-laravel/actions/runs/8425699518/job/23072364127#step:5:35

Seems to be purely down to one of the packages installed is causing PHPStan to fail to detect the interfaces implemented by a class. It's not PHPStan itself or the PHP Parser but working out which package is the problem is kind of a needle in the haystack.

**Reply by GeniJaho** [reply]
I've had this problem before. For the Laravel ecosystem classes to be properly recognized, we should add them to the stubs directory, otherwise, tests fail with no indication of what's happening. I hope this PR fixes it. Of course, I don't know why 😅.
https://github.com/driftingly/rector-laravel/pull/202

**Comment by driftingly** [comment]
Released [1.1.0](https://github.com/driftingly/rector-laravel/releases/tag/1.1.0)! Thanks for your work on this last release.
